### DISCUSSION TITLE
Cannot remove/hide "Snapper GUI (root)" from the app grid

### DISCUSSION THREAD

**Discussion by lucia-loves-teto** [SpiralLinux/SpiralLinux-project#404] [General]
I can't remove that app from my SpiralLinux Gnome app grid. Menulibre won't let me remove and I can't hide it using the App Hider extension.
![Captura de tela de 2024-02-25 10-20-35](https://github.com/SpiralLinux/SpiralLinux-project/assets/161218194/b7672fb6-c99e-410a-9c8b-5448329192a5)

**Comment by geckolinux** [comment]
Hi there, you would need to delete `/usr/share/applications/snapper-gui-root.desktop` .

**Reply by lucia-loves-teto** [reply]
Worked! Thanks =D
